### DISCUSSION TITLE
Is it possible to port the project to use method as described in the IsaacLab tutorial section?

### DISCUSSION THREAD

**Discussion by dustinvanstee** [abizovnuralem/go2_omniverse#29] [General]
Hi I was admiring the Go2 project you have built and tried to get it up and running in both IsaacSim 4.0/4.1 containers released by NVIDIA.  I had installed Ros2 dependencies and IsaacLab, but I didnt have much success.  I saw that IsaacLab has the isaaclab.sh script they recommend people to use and pass python to that.   Any plans to migrate to that method or get a version that runs in the Isaac Containers?  If interested I can share my errors I had.   see this site for reference -> https://isaac-sim.github.io/IsaacLab/source/tutorials/00_sim/launch_app.html

**Comment by abizovnuralem** [comment]
Hello, currently it is not tested with containers. Only clear Ubuntu 22.04 with IsaacSim 4.0. If you can optimize it for containers it will be great!
